packages/ra-ui-materialui/src/input/SelectInput.stories.tsx

```diff
@@ -1,24 +1,33 @@
-import * as React from 'react';
-import { Admin, AdminContext } from 'react-admin';
-import { Resource, required, useGetList, TestMemoryRouter } from 'ra-core';
-import polyglotI18nProvider from 'ra-i18n-polyglot';
-import englishMessages from 'ra-language-english';
+import CloseIcon from '@mui/icons-material/Close';
 import {
+    Button,
     Dialog,
-    DialogContent,
     DialogActions,
+    DialogContent,
+    DialogTitle,
+    IconButton,
     TextField,
-    Button,
 } from '@mui/material';
+import {
+    CreateBase,
+    Resource,
+    TestMemoryRouter,
+    required,
+    useGetList,
+} from 'ra-core';
+import polyglotI18nProvider from 'ra-i18n-polyglot';
+import englishMessages from 'ra-language-english';
+import * as React from 'react';
+import { Admin, AdminContext } from 'react-admin';
```

Comment:
Please do not import from `'react-admin'` as it will create a circular dependency